controllers/search.js

```diff
@@ -4,17 +4,27 @@ const Parent = require('../lib/hof-standalone');
 const api = require('../api');
 const helpers = require('../lib/helpers');
 const moment = require('moment');
+require('moment-round'); // add rounding support to moment
```

Comment:
Yes... Well unless you to hand-code:
```js
moment().set(0, 'hours').set(0, 'minutes').set(0, 'seconds').set(0, 'milliseconds')
```
...rather than simply:
```js
moment().floor(24, hours)
```